README.md

```diff
@@ -70,6 +71,12 @@ tr3 = CompositeTransform([tr2, tr1])
 print(tr3.map(coords))
 ```
 
+Caveats
```

Comment:
Doesn't seem like this belongs here. Maybe in the CordinateSystemGraph docstring?